src/ui/public/notify/notifier.js

```diff
@@ -310,48 +320,69 @@ Notifier.prototype._showFatal = function (err) {
  * @param  {Error|String} err
  * @param  {Function} cb
  */
-Notifier.prototype.error = function (err, cb) {
-  return add({
+Notifier.prototype.error = function (err, opts, cb) {
+  const overrideable = ['lifetime', 'icon'];
+  if (_.isFunction(opts)) {
+    cb = opts;
+    opts = {};
+  }
+
+  const config = _.assign({
     type: 'danger',
     content: formatMsg(err, this.from),
     icon: 'warning',
     title: 'Error',
     lifetime: Notifier.config.errorLifetime,
     actions: ['report', 'accept'],
     stack: formatStack(err)
-  }, cb);
+  }, _.pick(opts, overrideable));
```

Comment:
Well, the intention behind `_.assign / Object.assign` seemed clear enough to me that it didn't require extraction of a variable, but if you want to stick with it I'm OK with that.

In which case, I do think the name should be changed to `overridingOptions`, since it seems to me that `overrideable` implies the variable is or can be overridden, which doesn't reflect how it's being used.